websites/docs-smoke-test/gatsby-config.js

```diff
@@ -12,9 +12,9 @@ module.exports = {
     {
       resolve: '@commercetools-docs/gatsby-theme-docs',
       options: {
+        title: 'Docs Smoke Test',
```

Comment:
This is also duplicated

Reply:
Yea, not quite sure what the alternative would look like yet.

Reply:
We could "move" the `siteMetadata` as a theme plugin option, then spread the value in the theme `gatsby-config.js` as we have some defaults for `siteMetadata`.

This would be a breaking change though.

Reply:
I'm looking to override the title in the `setup` field of the plugin, where we have access to the siteMetadata.

https://www.gatsbyjs.org/docs/adding-an-rss-feed/#syntax-for-itunes-rss-blocks

But I see that the feed's global title is the same as the description so I'm trying to fix that now.

Reply:
@emmenko  we just refactored the way the options are read, no duplicates any more

Reply:
Thanks, I like the `setup` approach. The configuration has still an error though (see CI)